index.bs

```diff
@@ -3430,15 +3430,16 @@ engine.
     - Verify that `extraData` is set to the hash of |attToBeSigned| using the hash algorithm employed in "alg".
     - Verify that `attested` contains a `TPMS_CERTIFY_INFO` structure as specified in [[TPMv2-Part2]] section 10.12.3,
         whose `name` field contains a valid Name for |pubArea|,
-        as computed using the algorithm in the `nameAlg` field of |pubArea| using the procedure specified in [[TPMv2-Part1]]
-        section 16.
+        as computed by concatinating algorithm identifier from the signauture algorithm(for RS256 it's sha256) specified in TPM_ALG_ID, with hash of the `pubArea` field as specified in [[TPMv2-Part1]] section 16.
+    - Verify that `pubArea` contains TPMU_PUBLIC_ID that is set to newly generate public key as specified in [[TPMv2-Part2]] section 12.2.3.2.
     - Note that the remaining fields in the "Standard Attestation Structure" [[TPMv2-Part1]] 
         section 31.2, i.e., `qualifiedSigner`, `clockInfo` and `firmwareVersion` are ignored.
         These fields MAY be used as an input to risk engines.
 
     If |x5c| is present, this indicates that the attestation type is not [=ECDAA=]. In this case:
     - Verify the |sig| is a valid signature over |certInfo| using the attestation public key in |x5c| with the
         algorithm specified in |alg|.
+    - Verify certificate chain in |x5c| against attestationRootCertificates in Metadata Statement specified by AAGUID in authData.
```

Comment:
I'd argue this is already covered by steps 15 and 16 in [§7.1. Relying Party Operations: Registering a new credential](https://w3c.github.io/webauthn/#registering-a-new-credential), and doesn't need to be repeated here.

Reply:
perhaps steps 15 and 16 in [§7.1. Relying Party Operations: Registering a new credential](https://w3c.github.io/webauthn/#registering-a-new-credential) can be appropriately referenced here -- we can add anchors to individual steps if we wish -- see [step 13](file:///Users/jehodges/documents/work/standards/W3C/herrjemand/webauthn/index-herrjemand-clarifying-tpm-attestation-verification.html#authenticatorGetAssertion-return-values) in [S 6.2.3](file:///Users/jehodges/documents/work/standards/W3C/herrjemand/webauthn/index-herrjemand-clarifying-tpm-attestation-verification.html#op-get-assertion).